source/_components/nws.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: "NWS"
+description: "Instructions on how to integrate National Weather Service data within Home Assistant."
+date: 2019-05-02 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+featured: false
+#logo: nws.png
```

Comment:
Still missing a logo 😉

Reply:
I was waiting to get aligned with the other PR with NOAA as the name, but I think that the author suggested that NWS could be used.  I've read through the NWS disclaimer page one more time now.  The section I quoted above only applies to attribution and not the logo (at least it doesn't mention the logo).

The quote you gave clearly says you need no permission only if it is unaltered and part of original product.  Are you comfortable with that assertion?  The data is altered, in particular, changing the condition to match a predefined set of HA conditions.  The rest of alterations are more trivial like rounding, and unit conversions.  This doesn't seem to meet this strict criteria in my interpretation.